tests/foreman/ui/test_gpgkey.py

```diff
@@ -32,12 +40,10 @@ class GPGKey(UITestCase):
     """Implements tests for GPG Keys via UI"""
 
     @classmethod
-    def setUpClass(cls):  # noqa
-        org_attrs = entities.Organization().create_json()
-        cls.org_name = org_attrs['name']
-        cls.org_id = org_attrs['id']
-        cls.org_label = org_attrs['label']
-
+    def setUpClass(cls):
+        cls.key_content = read_data_file(VALID_GPG_KEY_FILE)
+        cls.key_path = get_data_file(VALID_GPG_KEY_FILE)
+        cls.organization = entities.Organization().create()
```

Comment:
I cannot set `cls.org`, as that attribute is already set by `UITestCase.setUp`. See [here](https://github.com/SatelliteQE/robottelo/blob/master/robottelo/test.py#L222).